### ISSUE TITLE
Not well result

### ISSUE THREAD

**Issue by Angus1996** [SSinyu/WGAN_VGG#3]
I run this code successfully.
But when i was testing，the pred result in saved fig seemed not right. The region that should be black was white???Is there anyone who know what's the probelm?

**Comment by stefenmax** [comment]
have u met this question like num_samples should be a positive integer value, but got num_samples=0
could u please tell me how to solve it thx

**Comment by yanglei0941212** [comment]
> I run this code successfully.
> But when i was testing，the pred result in saved fig seemed not right. The region that should be black was white???Is there anyone who know what's the probelm?

you should check the function **save_model** in **solver.py**,this is a pytorch program,so we should save model as 'WGANVGG_xx}iter.**pth**' not ‘WGANVGG_xx}iter.ckpt’,**because ckpt  file is for tensorflow**

**Comment by respect3** [comment]
> I run this code successfully. But when i was testing，the pred result in saved fig seemed not right. The region that should be black was white???Is there anyone who know what's the probelm?

Hi,could you please tell me your email address for better communication?And my email is [wangxiangcanwww@163.com](mailto:wangxiangcanwww@163.com)

**Comment by JiangGWei** [comment]
I successfully run the code, but the loss function I get from training is negative. And the prediction results are worse than LDCT. Any idea what the problem is?And my email is 779500270@qq.com

**Comment by chenjiachengzzz** [comment]
> I successfully run the code, but the loss function I get from training is negative. And the prediction results are worse than LDCT. Any idea what the problem is?And my email is [779500270@qq.com](mailto:779500270@qq.com)
Have you solved this problem?

**Comment by WhoYouAre7** [comment]
> > I successfully run the code, but the loss function I get from training is negative. And the prediction results are worse than LDCT. Any idea what the problem is?And my email is [779500270@qq.com](mailto:779500270@qq.com)
> > Have you solved this problem?

Have you solved this problem?

**Comment by EvelyneCalista** [comment]
> > > I successfully run the code, but the loss function I get from training is negative. And the prediction results are worse than LDCT. Any idea what the problem is?And my email is [779500270@qq.com](mailto:779500270@qq.com)
> > > Have you solved this problem?
> 
> Have you solved this problem?

Hi, have you solved the black image issue in inference?

**Comment by 123-c1** [comment]
请问 这个问题有人解决了吗，测试图像全是黑色的
